index.js

```diff
@@ -57,6 +59,18 @@ module.exports = function(gitPath) {
       }
 
       result.abbreviatedSha = result.sha.slice(0,10);
+
+      // Find tag
+      var packedRefsFilePath = path.join(gitPath, 'packed-refs');
+      if (fs.existsSync(packedRefsFilePath)) {
+        var packedRefsFile = fs.readFileSync(packedRefsFilePath, {encoding: 'utf8'});
+        var tagLine = packedRefsFile.split('\n').filter(function(l) {
+          return l.indexOf("refs/tags") > -1 && l.indexOf(result.sha) > -1;
```

Comment:
Tiny nit: I'm not a huge fan of single letter variables, can you change this to `line` or something?